src/main/java/com/easyrest/pages/MenuPage.java

```diff
@@ -1,140 +1,19 @@
 package com.easyrest.pages;
 
-import com.sun.jna.WString;
-import org.openqa.selenium.By;
+import com.easyrest.components.menuPage.Cart;
+import com.easyrest.components.menuPage.MenuItem;
+import com.easyrest.components.menuPage.OrderConfirmation;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-
-import java.util.List;
 
 public class MenuPage extends BasePage {
-
-
-    private final By watchMenu = By.xpath("//*[text() = 'Watch Menu']");
-    private final By addProduct = By.xpath("//button[@aria-label = 'Add to cart']");
-    private final By submitOrder = By.xpath("//*[text()='Submit order']");
-    private final By submitInWindowButton = By.xpath("//*[text()='Submit']");
-    private final By quantityItemField = By.xpath("//div[contains (@class ,'MuiDialogContent')]//tr[contains(@class,'MuiTableRow-root')][1]//input[@type = 'number' ]");
-    private final By showCartButton = By.xpath("//button [@aria-label = 'Show cart']");
-    private final By soupCategoryMenuItem = By.xpath("//span [text() ='Soup']");
-    private final By coctailsCategoryMenuItem = By.xpath("//span [text() ='Coctails']");
-    private final By salatsCategoryMenuItem = By.xpath("//span [text() ='Salats']");
-    private final By withBeerCategoryMenuItem = By.xpath("//span [text() ='With beer']");
-    private final By hotCategoryMenuItem = By.xpath("//span [text() ='Hot']");
-    private final By pizzaCategoryMenuItem = By.xpath("//span [text() ='Pizza']");
-    private final By meatCategoryMenuItem = By.xpath("//span [text() ='Meat']");
-    private final By pastaCategoryMenuItem = By.xpath("//span [text() ='Pasta']");
-    private final By bakeryCategoryMenuItem = By.xpath("//span [text() ='Bakery']");
-    private final By hardLiquerCategoryMenuItem = By.xpath("//span [text() ='Hard liquer']");
-    private final By deleteSecondItemButton = By.xpath("//div[contains(@class , 'MuiPaper-root')][2]//div[contains(@style ,'' )]//button [@aria-label ='Remove item']");
+    private Cart cart;
+    private MenuItem menuItem;
+    private OrderConfirmation orderConfirmation;
 
```

Comment:
Just check if you can use these private instance variables, maybe you need some methods